### DISCUSSION TITLE
v2.1 The "HYPER-Sequoia" Update! 🎧

### DISCUSSION THREAD

**Discussion by MultimediaLucario** [MultimediaLucario/Lenovo-ThinkPad-T480#28] [Announcements]
![repository-open-graph-template-sequoia](https://github.com/user-attachments/assets/d139f225-0d2a-4335-b9fc-e743faad0983)

# The HYPER-Sequoia Update!  💻🎧

Built on top of the previous release, it's time to enter the rave with the first new update since our (Daft Lucario's) B2B performance with Zanny Duko @ HYPERGLOW 2024 @ The House of Independents in Asbury Park, NJ (Hence the name HYPER-Sequoia)! This update contains the newly released OpenCore Bootloader version ```1.0.1```, new patches such as improved stability with the headphone jack's audio, and the most requested new feature, macOS Sequoia support! All of the very self-explanatory EFI folders include these new patches. So, go check them out! 😎🔥

**Full Changelog**: https://github.com/MultimediaLucario/Lenovo-ThinkPad-T480/compare/v2.0...v2.1

<hr /><em>This discussion was created from the release <a href='https://github.com/MultimediaLucario/Lenovo-ThinkPad-T480/releases/tag/v2.1'>v2.1 The "HYPER-Sequoia" Update! 🎧  </a>.</em>

**Comment by dgorbatko** [comment]
Works perfectly on T480S Sequoia except Bluetooth.
Wifi works with Heliport.
I only changed MAC and other numbers with ProperTree.

Does bluetooth works for you on T480 Sequoia?

**Reply by dgorbatko** [reply]
Ah and the awful touchpad bug has been fixed in your efi, there is no more delay between touch and key! So I finally can use it in my 3d apps without this delay!

**Reply by MultimediaLucario** [reply]
Yes. Bluetooth does work.

**Comment by fpassold** [comment]
Hello. Firstly, thank you for the work you are doing and for making EFIs available. I tested your two EFI versions for the recently released Sequoia. They work except for small details... I would like to know more specifically what Wifi/Bluetooth card your Thinkpad T480 uses. Mine is similar to yours, using an Intel AC8265 (WiFi/Bluetooth) and Intel I219-LM board (Ethernet). In addition to a Realtek ALC257 audio card. But I need to use the version with "AirportItlmw" support, because  I need to log into a WPA2 Enterprise WiFi network. That is, I cannot use the version with Heliport because I lose this functionality, which is necessary for me. Furthermore, I noticed that after the notebook goes into sleep mode, the audio comes back distorted (low volume, with a kind of reverberation, "echo"). Going into settings and making some audio adjustments sometimes fixes the problem (a slight inconvenience). But the main problem is with the Intel WiFi card. I have been searching on the internet for ways to activate "AirportItlwm" with an Intel card (in my case AC8265), but I am not having success. I appreciate tips/suggestions to try to overcome this problem.

**Reply by MultimediaLucario** [reply]
AirPortItlwm does not work with iServices just yet. I am trying to wait to see when it will work for Sonoma and I will drop an update when that happens.

**Reply by tetenc555** [reply]
Alternatively, you could buy a native broadcom card, i'm just waiting for my new BCM94360NG to arrive. I was using a BCM94360CS2 with my old Lenovo and it worked in my college's WPA2 Enterprise WiFi flawlessly

**Comment by fpassold** [comment]
Thanks Estêvão Fiorilo for the answer. 

I didn't understand which OpenIntelWireless kext files you intend to use with the Broadcom card so that it can connect to WPA2 Enterprise WiFi networks. Please confirm if you will use the combination: 1) "AirportItlwm.kext" or 2) "Itlwm.kext + Heliport" to make this Broadcom card work with WPA2 Enterprise.

 I thought just using "AirportItlwm.kext" allows WiFi WPA2 Enterprise connections.

My Thinkpad is equipped with an Intel AC 8265 WiFi card. I managed to run macOS Sequoia 1.1 plus this intel WiFi board using "Itlwm.kext" + “Heliport". I was able to connect to the WiFi network at home (WPA) but not at work (WPA2 Enterprise). But I was unable to activate Bluetooth in case of Sequoia. The Sequoia tries to activate Bluetooth but nothing happens. Sequoia seemed to run faster than Ventura (despite booting or logging in being a little slower).
Despite this I went back to Ventura 13.7 using "AirportItlwm.kext" and this way I can connect to the WPA2 Enterprise WiFi network and Bluetooth works with some issues (after notebook sleeps). Unfortunately, I need WAP2 Enterprise connection for my work.

I think the solution will be to wait for an "AirportItlwm.kext” update for Sonoma >= 14.4 or Sequoia >= 15.1. At least for those who prefer to continue with Intel WiFi cards and need a WPA2 Enterprise connection.

**Reply by MultimediaLucario** [reply]
I just dropped a new release
